.gitmodules

```diff
@@ -1,4 +1,10 @@
+[submodule "contrib/scons-build-tools"]
+	path = contrib/scons-build-tools
+	url = git@research.com:softwarecenter/scons-build-tools.git
+[submodule "contrib/googletest-fused"]
+	path = contrib/googletest-fused
+	url = git@research.com:softwarecenter/googletest-fused.git
 [submodule "contrib/outpost-core"]
 	path = contrib/outpost-core
-	url = git@github.com:DLR-RY/outpost-core.git
-	branch = master
+	url = git@research.com:pengting/outpost-core.git
+	branch = 502-master
```

Comment:
Probably want to keep it as is

Reply:
The original one has compilation problem with SCons in modules/support/ for my task. I will verify whether it happens to tasking framework by using original repo and master branch.